projects/storefrontlib/src/lib/product/product-outlets.model.ts

```diff
@@ -2,11 +2,6 @@ export enum ProductDetailOutlets {
   PAGE = 'PDP.PAGE',
   SUMMARY = 'PDP.SUMMARY',
   IMAGES = 'PDP.IMAGES',
-  TAB = 'PDP.TAB',
-  DESCRIPTION = 'PDP.DESCRIPTION',
```

Comment:
We need to maintain

DESCRIPTION = 'PDP.DESCRIPTION',
SPECIFICATIONS = 'PDP.SPECIFICATIONS',
REVIEWS = 'PDP.REVIEWS',
SHIPPING = 'PDP.SHIPPING',

for as long as we haven't finalised #1484

Reply:
ok, I moved it to separated enum dedicated for Tabs